coordinator-common/src/test/java/org/apache/kafka/coordinator/common/runtime/HdrHistogramTest.java

```diff
@@ -172,4 +178,39 @@ public void testHistogramDataReset() {
         assertEquals(numEventsInFirstCycle, hdrHistogram.count(now + maxSnapshotAgeMs));
         assertEquals(numEventsInSecondCycle, hdrHistogram.count(now + 1 + maxSnapshotAgeMs));
     }
+
+    @Test
+    public void testLatestHistogramRace() throws InterruptedException, ExecutionException {
```

Comment:
I'm not sure it's preferable. We don't ensure somehow that `latestHistogram()` will not return unfilled histogram - if there are no recordings, the histogram snapshot will be empty. I think this tests specifically threads racing to call `latestHistogram()`, so the current name seems well-fitting to me.

Reply:
I agree the term "unfilled" may not be appropriate but my intention was that the histogram snapshot will always contain recordings since MAX_SNAPSHOT_AGE (or older).

I like test names to be a bit more descriptive since it adds readability but i guess the existing name works.